lib/gc/Transforms/IterativeTilingAndFusion.cpp

```diff
@@ -543,7 +548,13 @@ LogicalResult iterativelyFuseProducerAndConsumerOfTiledOp(
   return success(numTiledOps > 1);
 }
 
-/// What is self tiled op compared with other fused op?
+/// This is a workaround to deal with LinalgXOp
+static bool isTilableLinalgXOp(Operation *op) {
+  return isa<linalgx::BatchReduceMatmulVnniOp, linalgx::MultiBatchMatmulOp,
```

Comment:
This is a workaround because `LinalgXOp` is not inherited from `TilingInterface` so far.